jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -140,6 +140,36 @@ pipeline {
                         }
                     }
                 }
+                stage('build-snapshot-macos-x64-tar') {
+                    agent {
+                        node {
+                            label 'Jenkins-Agent-MacOS12-X64-Mac1Metal-Multi-Host'
+                        }
+                    }
+                    tools {
+                        jdk dockerAgent.javaVersion
+                    }
+                    steps {
+                        script {
+                            buildManifest(
+                                componentName: "${COMPONENT_NAME}",
+                                inputManifest: "manifests/${INPUT_MANIFEST}",
+                                distribution: 'tar',
+                                snapshot: true
+                            )
+                            echo("Uploading min snapshots to S3")
```

Comment:
Say uploading macos min snapshot to S3. Better logging :)

Reply:
Done @gaiksaya